### PR TITLE
build and packaging script improvement for release

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: test/shared_test/test_ortops.cc [hunk 1]
```diff
@@ -303,11 +305,20 @@ TEST(ustring, tensor_operator) {
...
+    api->ReleaseStatus(status);
+  }
+
   Ort::CustomOpApi custom_api(*api);
 
   std::vector<int64_t> dim{2, 2};
-  api->CreateTensorAsOrtValue(allocator, dim.data(), dim.size(), ONNX_TENSOR_ELEMENT_DATA_TYPE_STRING, &tensor);
+  status = api->CreateTensorAsOrtValue(allocator, dim.data(), dim.size(), ONNX_TENSOR_ELEMENT_DATA_TYPE_STRING, &tensor);
+  ASSERT_TRUE(status == nullptr || api->GetErrorCode(status) == ORT_OK);
+  if (status != nullptr) {
```
Comment (wenbingl) [inline test/shared_test/test_ortops.cc:319]: it looks funny ...

File: includes/onnxruntime/onnxruntime_c_api.h [hunk 2]
```diff
@@ -49,7 +52,11 @@ extern "C" {
 #else
 #define ORT_EXPORT
 #endif
+#if defined(__MINGW32__)
```
Comment (snnn) [inline includes/onnxruntime/onnxruntime_c_api.h:55]: Do you need to support mingw32?

File: test/shared_test/test_ortops.cc (hunk 1 above)
Comment (snnn) [reply test/shared_test/test_ortops.cc:319]: ?

File: includes/onnxruntime/onnxruntime_c_api.h (hunk 2 above)
Comment (wenbingl) [reply includes/onnxruntime/onnxruntime_c_api.h:55]: Yes, some people asked that, and this small fixing can enable it.

File: test/shared_test/test_ortops.cc (hunk 1 above)
Comment (wenbingl) [reply test/shared_test/test_ortops.cc:319]: there is a warning if the returned status discard. I copied this code from somewhere, but I just noticed it is not good.

File: includes/onnxruntime/onnxruntime_c_api.h (hunk 2 above)
Comment (snnn) [reply includes/onnxruntime/onnxruntime_c_api.h:55]: You may consider send a PR to the main repo.

**Review by snnn — APPROVED** [review]

File: test/shared_test/test_ortops.cc (hunk 1 above)
Comment (snnn) [reply test/shared_test/test_ortops.cc:319]: It looks good to me.

**Pull request merged**